network/init.sh

```diff
@@ -79,6 +83,17 @@ sed -i -e 's#"tcp://localhost:1317"#"tcp://0.0.0.0:'"$RESTPORT"'"#g' "$CHAIN_DIR
 sed -i -e 's#"tcp://0.0.0.0:1317"#"tcp://0.0.0.0:'"$RESTPORT"'"#g' "$CHAIN_DIR/config/app.toml"
 sed -i -e 's#":8080"#":'"$ROSETTA_1"'"#g' "$CHAIN_DIR/config/app.toml"
 
+# NOTE: we provide prometheus_server_address but it does not matter, since we don't use it, just need non-empty value
```

Comment:
what do you mean we don't user it? I see a `true` default value above for `ORACLE_METRICS_ENABLED`. am I missing something?

Reply:
yes, metrics enabled, but they're also having a "prometheus_server_address" value, that is for starting a second prometheus server (i don't know what for), and it should be non-empty because of validation.
But it's not used in our scenario, so we provide some non empty value here